instrumentation/github.com/bradfitz/gomemcache/attributes.go

```diff
@@ -0,0 +1,55 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gomemcache
+
+import (
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/api/standard"
+)
+
+const (
+	operationAdd            = "add"
+	operationCompareAndSwap = "cas"
+	operationDecrement      = "decr"
+	operationDelete         = "delete"
+	operationDeleteAll      = "delete_all"
+	operationFlushAll       = "flush_all"
+	operationGet            = "get"
+	operationIncrement      = "incr"
+	operationPing           = "ping"
+	operationReplace        = "replace"
+	operationSet            = "set"
+	operationTouch          = "touch"
+
+	mamcacheDBSystemValue = "memcached"
+
+	memcacheDBItemKeyKeyName kv.Key = "db.memcached.itemKey"
+)
+
+func memcacheDBSystem() kv.KeyValue {
+	return standard.DBSystemKey.String(mamcacheDBSystemValue)
+}
+
+func memcacheDBOperation(opName string) kv.KeyValue {
+	return standard.DBOperationKey.String(opName)
+}
+
+func memcacheDBItemKeys(itemKeys ...string) []kv.KeyValue {
+	kvs := make([]kv.KeyValue, 0, len(itemKeys))
+	for _, ik := range itemKeys {
+		kvs = append(kvs, memcacheDBItemKeyKeyName.String(ik))
```

Comment:
Having multiple values for a metric label will mean that [the last one wins](https://github.com/open-telemetry/opentelemetry-go/blob/b7df5543e398789ae2f9ae75c0868a92a55d7fe0/api/label/set.go#L280) and [for attributes, the first ones are "more likely" to be dropped](https://github.com/open-telemetry/opentelemetry-go/blob/b7df5543e398789ae2f9ae75c0868a92a55d7fe0/sdk/trace/attributesMap.go#L25-L26).

Is that acceptable here, or did we want to try and join them into a single attribute?

Reply:
That's a good catch, my intention was to include all item keys, in case an operation occasionally includes multiple items. What do you think about making the KV into `Array` in case of multiple items? (My suggestion in the latest commit)